project.md

```diff
@@ -0,0 +1,48 @@
+#### Name: Site Scanner
+
+
+#### Elevator Pitch:  
+
+Test out if a site is reputable by requesting the site's repuatation from Auth0, Google, and get a screenshot of the site just by entering a link.  The user can enter a site, see an image and get a "yes or no" this is okay.
+
+#### Wireframe:
+![alt text](sitescanner.png "SiteScanner")
+
+
+#### Dependencies:
+APIs:
+www.thum.io/
+https://auth0.com/signals/ip
+https://developers.google.com/safe-browsing/v4/request-frequency
```

Comment:
Something to note with these APIs are that they all use API keys that _shouldn't_ be exposed in the React app at all. If you already started incorporating them, I would just make sure you don't deploy API keys that are attached to some sort of subscription/paid API.